src/main/java/net/sigmalab/maven/plugin/jira/CreateNewVersionMojo.java

```diff
@@ -98,7 +98,7 @@ boolean versionAlreadyExists(Iterable<Version> remoteVersions, String newDevVers
      * @return
      */
     private String computeVersionName() {
```

Comment:
**Remove the literal "false" boolean value.**

Redundant Boolean literals should be removed from expressions to improve readability.